### DISCUSSION TITLE
System.DllNotFoundException

### DISCUSSION THREAD

**Discussion by zombers** [dotnet/maui#16063] [Q&A]
If my maui project needs to reference an lsllib project, and the lsllib project has an lsl.dll that needs to be referenced. My maui project runs normally on the windows version, but when compiled into maui.android, it prompts that lsl.dll cannot be found. What should I do?

**Comment by Xyncgas** [comment]
> If my maui project needs to reference an lsllib project, and the lsllib project has an lsl.dll that needs to be referenced. My maui project runs normally on the windows version, but when compiled into maui.android, it prompts that lsl.dll cannot be found. What should I do?

Copy the dll from somewhere maybe from the app that's running on windows

**Reply by zombers** [reply]
I don’t know how to copy this dll to the corresponding path of the android project. I have set lsl.dll to copy if newer in the lilllib project.

**Reply by Xyncgas** [reply]
Idk either, can you do it by hand, even though it's inconvenient I bet you can find that dll in the publish folder for your windows app and you can copy it to the android publish folder where all the other dlls are at

**Comment by stevexuzeta** [comment]
My maui project runs wrong even on the windows version,  but it runs well when I import dll in dotnet console project. I have no idea, can you show me how to import it well on the windows version in maui project? Are there any tips?
